desktop/fileManager.js

```diff
@@ -30,6 +30,19 @@ class FileManager {
     });
   };
 
+  static openWalletBackupDirectory = ({ browserWindow, event }) => {
```

Comment:
this will open dialog modal for selecting file or directory, what you actually need is this function call
**shell.showItemInFolder(fullPath)**
that will open finder/explorer window and select the file

Reply:
ok